subsys/shell/backends/shell_rtt.c

```diff
@@ -25,7 +31,8 @@ SHELL_DEFINE(shell_rtt, CONFIG_SHELL_PROMPT_RTT, &shell_transport_rtt,
 
 LOG_MODULE_REGISTER(shell_rtt, CONFIG_SHELL_RTT_LOG_LEVEL);
 
-static bool rtt_blocking;
+static bool panic_mode;
```

Comment:
This variable is never set?

Reply:
^^ @nordic-krch

Reply:
It seems that api for settting panic mode is missing i shell_transport_api.  In NCS setup this causes an attempt to lock mutex ISR context, when entering from log_panic

Reply:
I guess this was never acted upon, since the issue is still present in 3.7.0.